discojs-web/src/memory/memory.ts

```diff
@@ -9,11 +9,11 @@
 import { Map } from 'immutable'
 import * as tf from '@tensorflow/tfjs'
 
-import type { Path, Model, ModelInfo, ModelSource } from '@epfml/discojs'
+import type { Model, ModelInfo, ModelSource } from '@epfml/discojs'
 import { Memory, models } from '@epfml/discojs'
 
 export class IndexedDB extends Memory {
-  override getModelMemoryPath (source: ModelSource): Path {
+  override getModelMemoryPath (source: ModelSource): string {
```

Comment:
Why change Path to string?

Reply:
Ah I see it's to inline the type

Reply:
it's called `Path` but isn't really, it is used an ID for the models rather than a filesystem path. I found it confusing (which is kinda against the idea of having a dedicated type).